packages/assets/src/loader/parsers/loadTexture.ts

```diff
@@ -46,7 +46,7 @@ export const loadTextures = {
         const tempURL = url.split('?')[0];
         const extension = tempURL.split('.').pop();
 
-        return validImages.includes(extension);
+        return url.includes('data:image') || validImages.includes(extension);
```

Comment:
This could return a false-positive with "data:image/svg+xml", we probably need something stricter that considers the valid types too. 

Also, includes() is a little too ambiguous. Should probably use indexOf or RegExp instead.

Reply:
> This could return a false-positive with "data:image/svg+xml", we probably need something stricter that considers the valid types too.
> 
> Also, includes() is a little too ambiguous. Should probably use indexOf or RegExp instead.

I have thought it like:

`    test: function (url) {
        var tempURL = url.split('?')[0];
        var extension = tempURL.split('.').pop();

        var isValidBase64;
        for (var i = 0; i < validImages$4.length; i++) {
            if (url.indexOf('data:image/' + validImages$4[i]) > -1) {
                isValidBase64 = true;
                break;
            }
        }

        return isValidBase64 || validImages$4.includes(extension);
    }`

and this snippet is working well. If it is okay I can create a PR for it.

Reply:
This is better! The only change I would make is === 0 instead of > -1.